app/src/main/java/fr/free/nrw/commons/settings/SettingsFragment.java

```diff
@@ -68,40 +61,42 @@ public void onCreate(Bundle savedInstanceState) {
         EditTextPreference authorName = (EditTextPreference) findPreference("authorName");
         authorName.setEnabled(prefs.getBoolean("useAuthorName", false));
         useAuthorName.setOnPreferenceChangeListener((preference, newValue) -> {
-            authorName.setEnabled((Boolean)newValue);
+            authorName.setEnabled((Boolean) newValue);
             return true;
         });
 
         final EditTextPreference uploadLimit = (EditTextPreference) findPreference("uploads");
-        int uploads = prefs.getInt(Prefs.UPLOADS_SHOWING, 100);
-        uploadLimit.setText(uploads + "");
-        uploadLimit.setSummary(uploads + "");
+        int currentUploadLimit = prefs.getInt(Prefs.UPLOADS_SHOWING, 100);
+        uploadLimit.setText(Integer.toString(currentUploadLimit));
```

Comment:
Some explanation comment about  "100" would be nice.

Reply:
Agreed, will fix. 100 is default/fallback setting.

Reply:
I've fixed this, and also the arbitrary "500" limit too.